cernopendata_client/searcher.py

```diff
@@ -205,12 +208,12 @@ def get_files_list(
             else:
                 files_list_expanded.append(file_)
         files_list = files_list_expanded
-    if protocol == "http":
+    if searcher_protocol == "http":
         files_list = [
             (file_[0].replace(SERVER_ROOT_URI, server), file_[1], file_[2])
             for file_ in files_list
         ]
-    elif protocol == "https":
+    elif searcher_protocol == "https":
```

Comment:
Here is one situation I found that is not working:

```console


$ cernopendata-client get-file-locations --recid 5500 --server https://opendata.cern.ch --protocol xrootd | head -3
https://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/BuildFile.xml
https://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/HiggsDemoAnalyzer.cc
https://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/List_indexfile.txt
```

The value of `protocol` is not respected when `server` starts with `https`.

Can you please amend and add a test case?
```